deployment/network-operator/README.md

```diff
@@ -165,7 +169,26 @@ devicePlugin:
   deploy: true
   reources:
     - name: rdma_shared_device_a
-      devices: [enp1, enp2]
+      ifNames: [enp1, enp2]
     - name: rdma_shared_device_b
-      devices: [ib0]
+      ifNames: [ib0]
+```
+
+#### Example 3
+Network Operator deployment with the default version of OFED and NV Peer mem driver, RDMA device
+plugin with two RDMA resources, mapped to Mellanox ConnectX-5.
```

Comment:
The description does not match the yaml below

also can we have example 3 to deploy only device plugin ?


from my comment above
> you can have this example demonstrate a network operator deployment of device plugin only